code/game/gamemodes/blob/powers.dm

```diff
@@ -249,4 +249,22 @@
 	src << "<i>Factory Blobs</i> are blobs that spawn blob spores which will attack nearby enemies. This type of blob must be placed near node blobs or your core to work."
 	src << "<i>Node Blobs</i> are blobs which grow, like the core. Like the core it can activate resource and factory blobs."
 	src << "<b>In addition to the buttons on your HUD, there are a few click shortcuts to speed up expansion and defense.</b>"
-	src << "<b>Shortcuts:</b> Click = Expand Blob <b>|</b> Middle Mouse Click = Rally Spores <b>|</b> Ctrl Click = Create Shield Blob <b>|</b> Alt Click = Remove Blob"
\ No newline at end of file
+	src << "<b>Shortcuts:</b> Click = Expand Blob <b>|</b> Middle Mouse Click = Rally Spores <b>|</b> Ctrl Click = Create Shield Blob <b>|</b> Alt Click = Remove Blob"
+
+/datum/action/innate/blob_burst
+	name = "Burst"
+	button_icon_state = "blob"
+	background_icon_state = "bg_alien"
+
+/datum/action/innate/blob_burst/CheckRemoval()
+	if(ticker.mode.name != "blob" || !ishuman(owner))
```

Comment:
:-1:

blob bursting needs to be set up to function on assignment of being a blob anyways so that it isn't hardcoded into the gamemode, so a player can be made a blob by an admin and get time to position themselves